firstvoices/backend/tests/test_apis/test_import_job_confirm_api.py

```diff
@@ -26,6 +26,16 @@ def get_expected_response(self, instance, site):
         # Not required for this endpoint
         return {}
 
+    def create_import_job(self, site, status=None):
+        return ImportJobFactory(
+            site=site,
+            data=factories.FileFactory(
+                content=get_sample_file("import_job/all_valid_columns.csv", "text/csv")
+            ),
+            validation_status=JobStatus.COMPLETE,
```

Comment:
Suggestion: Lets add the `validation_status` as a param as well, and make the `JobStatus.Complete` as the default.